spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java

```diff
@@ -66,6 +69,11 @@
 	private boolean failOnError = false;
 
 	/**
+	 * Flag to determine whether labeled directories should be ignored.
+	 */
+	private boolean ignoreLabeledDirectories = false;
```

Comment:
This will work as `spring.cloud.config.server.native.ignoreLabeledDirectories=true`, but I don't like the name.

Maybe `addLabelLocations` default to true?